src/oscar/apps/catalogue/views.py

```diff
@@ -131,7 +132,7 @@ def get(self, request, *args, **kwargs):
         except InvalidPage:
             # Redirect to page one.
             messages.error(request, _('The given page number was invalid.'))
-            return redirect('catalogue:index')
```

Comment:
I don't think we should change this particular URL within the catalogue app. The catalogue app has it's own index URL defined, and we should respect that - it's possible that the home page of a project is different from the catalogue root.

Reply:
Ok! thank you @solarissmoke, i'll update the PR.

Reply:
done!